colossalai/shardformer/flashattention/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .flash_attention_opt import FlashAttention
+
+__all__ = ["FlashAttention"]
```

Comment:
`flashattention` should not be put as a first-level directory for `shardformer`. Instead, it should be categorized into the `layers` folder. Meanwhile, take care of your naming convenction, it should be `flash_atttention` for better readability.